src/PointwiseFunctions/AnalyticSolutions/GrMhd/SmoothFlow.cpp

```diff
@@ -146,18 +146,18 @@ SmoothFlow::variables(
   Scalar<DataType> specific_internal_energy = std::move(
       get<hydro::Tags::SpecificInternalEnergy<DataType>>(variables<DataType>(
           x, t, tmpl::list<hydro::Tags::SpecificInternalEnergy<DataType>>{})));
-  get(specific_internal_energy) *= adiabatic_exponent_;
+  get(specific_internal_energy) *= adiabatic_index_;
   get(specific_internal_energy) += 1.0;
   return {std::move(specific_internal_energy)};
 }
 
 bool operator==(const SmoothFlow& lhs, const SmoothFlow& rhs) noexcept {
   // there is no comparison operator for the EoS, but should be okay as
-  // the adiabatic_exponents are compared
+  // the adiabatic_indexs are compared
```

Comment:
indices

Reply:
But then it no longer matches the variable name, which is what I thought you had originally intended. I could do `'adiabatic_index's`, is that okay?

Reply:
Ah, I see what you mean - yes I would go with using the `s
```suggestion
  // the `adiabatic_index`s are compared
```

Reply:
since this is a blocker on the critical path let's defer a minor change for now, if that's alright with you.